### PR TITLE
Add and update

### PR DESCRIPTION
**Pull request by breadcat**
Add retroarch, update youtube-dl entry

### CODE REVIEW

File: just-install.json [hunk 1]
```diff
@@ -2109,7 +2117,7 @@
             "{{.PROGRAMFILES_X86}}\\youtube-dl\\youtube-dl.exe"
           ]
         },
-        "x86": "https://github.com/rg3/youtube-dl/releases/download/2018.05.01/youtube-dl.exe"
+        "x86": "https://github.com/rg3/youtube-dl/releases/download/{{.version}}/youtube-dl.exe"
```
Comment (geek1011) [inline just-install.json]: This will get reset next time just-install-updater-go runs.

**Comment by geek1011** [comment]
Thanks!

You can also add a rule to just-install-updater-go to automatically update retroarch.

**Comment by breadcat** [comment]
Thanks for merging!

I've had a look at the updater repo but just can't seem to fathom the syntax. I'm more bash scripts and python than javascript and webdev. I'd rather leave something be than merge something broken.
I check by every now and again and update the packages I've added.

**Comment by geek1011** [comment]
@breadcat I think you were looking at the old updater. The new one is written in go and has a simpler syntax. See https://github.com/just-install/just-install-updater-go/blob/master/jiup/rules/rules.go .

I can add a rule if you want.

**Pull request merged**